tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DynamicConfigurationTests.cs

```diff
@@ -166,8 +164,13 @@ private async Task UpdateAndValidateConfig(MockTracerAgent agent, LogEntryWatche
 
             var request = await agent.SetupRcmAndWait(Output, new[] { ((object)new { lib_config = config }, DynamicConfigurationManager.ProductName, fileId) });
 
+            // copy the byte array and reverse the bytes to create a BitArray with the correct order
+            var capabilityBytes = new byte[request.Client.Capabilities.Length];
+            Array.Copy(request.Client.Capabilities, capabilityBytes, request.Client.Capabilities.Length);
+            Array.Reverse(capabilityBytes);
```

Comment:
How has this been working up to this point?! 🙈

Reply:
Pure _coincidence_ that the flags we check for here were set in either direction until I added 29 for remote sampling rules in #5453. This test needs to be improved (e.g. assert more flags that are also set that we don't check here, and assert that the rest are false).